Lib/typing.py

```diff
@@ -1786,7 +1786,7 @@ def lookup_name(mapping: Mapping[KT, VT], key: KT, default: VT) -> VT:
 
     @_tp_cache
     def __class_getitem__(cls, params):
-        """Parameterizes a generic class.
+        """Parametrizes a generic class.
```

Comment:
Similar to the other PR, there is a "parameterize" two lines below and we should stay consistent.